src/Transloadit.ts

```diff
@@ -626,6 +626,38 @@ export class Transloadit {
     return { signature, params: jsonParams }
   }
 
+  /**
+   * Construct a signed Smart CDN URL. See https://transloadit.com/docs/topics/signature-authentication/#smart-cdn.
+   */
+  getSignedSmartCDNUrl(opts: SmartCDNUrlOptions): string {
+    if (opts.workspace == null) throw new TypeError('workspace is required')
+    if (opts.template == null) throw new TypeError('template is required')
+    if (opts.input == null) throw new TypeError('input is required')
+
+    const workspaceSlug = encodeURIComponent(opts.workspace)
+    const templateSlug = encodeURIComponent(opts.template)
+    const inputField = encodeURIComponent(opts.input)
+    const expiresIn = opts.expiresIn || 1 * 60 * 60 * 1000 // 1 hour
+
+    // Convert urlParams to Record<string, string>
+    const stringifiedParams: Record<string, string> = {}
+    for (const [key, value] of Object.entries(opts.urlParams || {})) {
+      stringifiedParams[key] = `${value}`
+    }
```

Comment:
what's the use case for allowing arbitrary data? if someone puts for example an object it will become `[object Object]` which is probably a bug. should we require values to be strings?